index.html

```diff
@@ -378,7 +377,7 @@ <h3>Normative Specifications</h3>
             </p>
             <p class="draft-status">
 <b>Draft State</b> to be adopted from: <a
-href="https://w3c-ccg.github.io/ld-security/">Linked Data Security 1.0</a>, eds.
+href="https://w3c-ccg.github.io/ld-proofs/">Linked Data Proofs 1.0</a>, eds.
```

Comment:
I meant to make this change... we need to combine the proofs/signatures specs... they're separate right now and need to be merged. If you were ok with the name "Linked Data Security 1.0", I was going to start the process of merging the specs.

Reply:
I do not mind the title, but if we go to W3C we need a live link. Can you do something quickly, or is it o.k. to merge it as is, and make the change _before_ the formal vote starts?

Reply:
I'm fine w/ merging as-is and then making the change before the formal vote.